### PR TITLE
chore: Add typescript

### LINKED ISSUE #73: Using cy.api command in UI tests also displays API request/response
I am trying to use the cy.api command in a base class like this:

```js  
protected post(request: ApiRequest): Cypress.Chainable<Cypress.Response> {
    return cy.api({
      method: HttpMethod.POST,
      url: this.buildUrl(request),
      failOnStatusCode: false,
      auth: this.setAuth(request.token),
      body: request.payload
    }).as(API_RESPONSE_ID);

  }
````

But I am using the same base class both for API and UI testing. 
Is there a way to hide the request/response details in the test runner?

### PR DESCRIPTION
**Pull request by LeJeanbono**
Should fix #73 with the new option `apiDisplayRequest`

### CODE REVIEW

**Comment by bahmutov** [comment]
Really nice

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Add the following line to your Cypress support file
 
 ```js
 // usually cypress/support/index.js
-import '@bahmutov/cy-api/support'
+import '@bahmutov/cy-api'
```
Comment (LeJeanbono) [inline README.md:25]: We have a breaking change here @bahmutov , is it ok ?

**Comment by bahmutov** [comment]
Yes. It is just a number and honestly the change to move the versions is easy

**Pull request merged**